### PR TITLE
feat(1-cms): Add Product Carousel section

### PR DESCRIPTION
**Pull request by ncuillery**
Now with product carousel:

### CODE REVIEW

File: lib/product_list/cubits/product_list_cubit.dart [hunk 1]
```diff
@@ -28,4 +29,18 @@ class ProductListCubit extends Cubit<ProductListState> {
       ));
     }
   }
+
+  Future<void> loadProductsfromIds({required List<String> productIds}) async {
+    for (var id in productIds) {
+      GetIt.instance
+          .get<ProductApi>()
+          .fetchProduct(productId: id, fields: LIST_PARAMS)
+          .then((value) {
```
Comment (stephenrichter) [inline lib/product_list/cubits/product_list_cubit.dart:38]: First time I'm seeing then vs. await, is this on purpose?

File: lib/home/home_page_content.dart [hunk 2]
```diff
@@ -39,6 +45,49 @@ class HomePageContent extends StatelessWidget {
...
+                      ),
+                      Text(product.price!.formattedValue),
+                    ],
+                  );
+                } else {
+                  return Container(
+                    height: 300,
+                    width: 400,
+                    child: const Center(
+                      child: CircularProgressIndicator(),
```
Comment (stephenrichter) [inline lib/home/home_page_content.dart:83]: Hmm.. if we can't find the product, CachedImage should handle that. Not sure we need this additional custom widget?

File: lib/product_list/cubits/product_list_cubit.dart (hunk 1 above)
Comment (ncuillery) [reply lib/product_list/cubits/product_list_cubit.dart:38]: Right, so initially I went with `Future.wait()` but it waits for ALL the requests to be completed to save ALL products in state.

I didn't want one request to block the display of the whole carousel so I went with this "fire and forget" implementation.

File: lib/home/home_page_content.dart (hunk 2 above)
Comment (ncuillery) [reply lib/home/home_page_content.dart:83]: The loader inside CachedImage is shown when the image is loading. This one is shown when the product is loading 🙂

Comment (ncuillery) [reply lib/home/home_page_content.dart:83]: I admit that I could be nicer though, I can create a shimmer widget

**Pull request merged**